src/components/transport_manager/src/android/local_socket_sender.cc

```diff
@@ -24,6 +24,7 @@ LocalSocketSender::LocalSocketSender(OnDataSentCallback&& sent_callback, OnConne
     , message_queue_()
     , connected_callback_(connected_callback)
     , sent_callback_(sent_callback)
+    , transport_name_("Unassigned")
```

Comment:
@VadymLuchko I think it's better to pass something like `LoggerDelegate&& callback` which accepts `std::string` as an input param. Then you don't need to store the copy of the transport name from another place. In the callback, you can access `controller_` and get the actual transport name

Reply:
@VadymLuchko please make it for both sender and receiver classes